app/views/devise/sessions/new.html.erb

```diff
@@ -1,12 +1,15 @@
+
+
 <div class="prettyform">
-  <div class="container" id="signin">
+  <div class="container con-format" id="signin">
```

Comment:
What's the `.con-format` class for? I don't see it referenced anywhere, and removing it doesn't seem to have any effect. Maybe it's no longer needed, and can be removed?